plugins/microsoft_defender_incidents/plugin.spec.yaml

```diff
@@ -0,0 +1,655 @@
+    required: true
+    example: 5cdad72f-c848-4df0-8aaa-ffe033e75d57
+types:
+  entity:
+    entityType:
+      title: Entity Type
+      description: Entities that have been identified to be part of, or related to a given alert, where possible values are - User, Ip, Url, File, Process, MailBox, MailMessage, MailCluster, and Registry
+      type: string
+    sha1:
+      title: SHA1
+      description: Available if entityType is File, containing file hash for alerts associated with a file or process
+      type: string
+      required: false
+    sha256:
+      title: SHA256
+      description: Available if entityType is File, containing file hash for alerts associated with a file or process
+      type: string
+      required: false
+    fileName:
+      title: File Name
+      description: Available if entityType is File, containing file name for alerts associated with a file or process
+      type: string
+      required: false
+    filePath:
+      title: File Path
+      description: Available if entityType is File, containing file path for alerts associated with a file or process
+      type: string
+      required: false
+    processId:
+      title: Process ID
+      description: Available if entityType is Process, containing identifier of the process
+      type: string
+      required: false
+    processCommandLine:
+      title: Process Command Line
+      description: Available if entityType is Process, containing command line of a process
+      type: string
+      required: false
+    processCreationTime:
+      title: Process Creation Time
+      description: Available if entityType is Process, containing creation time of a process
+      type: date
+      required: false
+    parentProcessId:
+      title: Parent Process ID
+      description: Available if entityType is Process, containing identifier of process parent
+      type: integer
+      required: false
+    parentProcessCreationTime:
+      title: Parent Process Creation Time
+      description: Available if entityType is Process, containing creation time of process parent
+      type: date
+      required: false
+    ipAddress:
+      title: IP Address
+      description: Available if entityType is IP, containing IP address associated with network event
+      type: string
+      required: false
+    url:
+      title: URL
+      description: Available if entityType is URL, containing URL for alerts associated to network events
+      type: string
+      required: false
+    accountName:
+      title: Account Name
+      description: Available if entityType is User, containing user account name
+      type: string
+      required: false
+    domainName:
+      title: Domain Name
+      description: Available if entityType is User, containing domain name
+      type: string
+      required: false
+    userSid:
+      title: User SID
+      description: Available if entityType is User, containing user SID
+      type: string
+      required: false
+    aadUserId:
```

Comment:
Is AAD an acronym? Something Active Directory? Would be helpful in the parameter's description. And if it is an acronym, please capitalize it like AAD.